skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -134,6 +134,8 @@ def marching_cubes_lewiner(volume, level=None, spacing=(1., 1., 1.),
         if level < volume.min() or level > volume.max():
             raise ValueError("Surface level must be within volume data range.")
     # spacing
+    if isinstance(spacing, np.ndarray):  # it should be a tuple
```

Comment:
Isn't it simpler to case to ndarray instead?

Reply:
possible, I was just looking for the smallest change to get the functionality

Reply:
I mean `spacing` could be a list as well. I think it is better to be more generic. Good catch, anyway!

Reply:
sure, it did work with lists before (but for the wrong reasons), I'll fix it